### DISCUSSION TITLE
ToolJet database table join and complex SQL queries

### DISCUSSION THREAD

**Discussion by danddinh** [ToolJet/ToolJet#6546] [Q&A] [answered]
ToolJet database operations seem limited to CRUD only:
![image](https://github.com/ToolJet/ToolJet/assets/116571017/f31bb6cb-c7a8-442d-83be-88aaa09b70f4)

Sometimes doing complex SQL queries is rather neccessary, eg. table join.

Is it possible to do custom SQL on ToolJet db?

**Comment by withshubh** [comment, marked as answer]
Hey @dandeancook - Currently, we do not have plans to incorporate custom SQL writing. However, we intend to offer a low-code solution for performing joins.
